src/test/java/org/broadinstitute/hellbender/tools/spark/validation/CompareBaseQualitiesIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.hellbender.tools.spark.validation;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class CompareBaseQualitiesIntegrationTest extends CommandLineProgramTest {
+    @Test
+    public void test() throws Exception {
+        final String resourceDir = getTestDataDir() + "/spark/validation/";
+
+        final File firstBam = new File(resourceDir, "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam");
+        final File secondBam = new File(resourceDir, "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam");
+
+        ArgumentsBuilder args = new ArgumentsBuilder();
+        args.add("--" + StandardArgumentDefinitions.INPUT_LONG_NAME);
+        args.add(firstBam.getCanonicalPath());
+        args.add("--" + "I2");
+        args.add(secondBam.getCanonicalPath());
+        args.add("--" + "cd");
+        args.add("true");
+        this.runCommandLine(args.getArgsArray());
```

Comment:
we need a proper test for it. So I think the tool should output the matrix and the summary to a file and we'd compare that files here